packages/data-explorer/tests/e2e/specs/test.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
   },
   'should display custom card': browser => {
     browser
-      .url(process.env.VUE_DEV_SERVER_URL + 'TableWithCustomCard')
+      .url(process.env.VUE_DEV_SERVER_URL + '#/TableWithCustomCard')
```

Comment:
i would think you would need to update the mock backend in the vue.config for this to work

Reply:
> i would think you would need to update the mock backend in the vue.config for this to work

guess not: only used for data requests